### PR TITLE
[Manager] Adapt to manger product prototype

### PR DESCRIPTION
**Pull request by weajun**
## Proposed changes

1. Support recording task status when installing a cluster
2. Integrated user permission verification, etc.
3. Use JDBC to add BE nodes
4. Increase the installation progress query, log query and other interfaces
4. Modify the interface document
5. Optimize the agent log code

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by morningman** [comment]
cc @songchuangyuan 
And please leave a message when this PR is ready to merge

File: manager/dm-server/src/main/java/org/apache/doris/stack/dao/AgentRoleRepository.java [hunk 1]
```diff
@@ -24,19 +24,17 @@
 
 import java.util.List;
 
-/**
- * Copyright (C) 2020 Baidu, Inc. All Rights Reserved.
```
Comment (hf200012) [inline manager/dm-server/src/main/java/org/apache/doris/stack/dao/AgentRoleRepository.java:28]: remove

File: manager/dm-server/src/main/java/org/apache/doris/stack/dao/AgentRepository.java [hunk 2]
```diff
@@ -24,13 +24,6 @@
 
 import java.util.List;
 
-/**
- * Copyright (C) 2020 Baidu, Inc. All Rights Reserved.
```
Comment (hf200012) [inline manager/dm-server/src/main/java/org/apache/doris/stack/dao/AgentRepository.java:28]: remove

**Review by hf200012 — COMMENTED** [review]
remove baidu copyright

File: manager/dm-agent/src/main/java/org/apache/doris/manager/agent/api/LogController.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+import java.util.Map;
+
+/**
+ * fe be log
+ **/
+@RestController
+@RequestMapping("/log")
+public class LogController {
+
+    private static final Logger LOG = LogManager.getLogger(LogController.class);
```
Comment (songchuangyuan) [inline manager/dm-agent/src/main/java/org/apache/doris/manager/agent/api/LogController.java:49]: Lombok has been introduced into the project. You can print logs directly using the lombok.extern.slf4j.slf4j tag

Comment (JNSimba) [reply manager/dm-agent/src/main/java/org/apache/doris/manager/agent/api/LogController.java]: thanks , already modified.

**Comment by songchuangyuan** [comment]
> cc @songchuangyuan And please leave a message when this PR is ready to merge

Codes can be merged

**Pull request merged**